src/controllers/flashcardsController.ts

```diff
@@ -0,0 +1,30 @@
+import { Request, Response } from "express"
+
+import Flascard from "../models/flashcardModel"
+import asyncHandler from "express-async-handler"
+
+//POST /flashcards
+export const createFlashcard = asyncHandler(
+  async (req: Request, res: Response) => {
+    if (!req.body.front || !req.body.back) {
+      res.status(400)
```

Comment:
**nitpick:** It would be more proper to chain this error response in `res`. 

```js
res.status(400).send("To add a card, you need the texts on the front and back sides")
return;
```